setup.py

```diff
@@ -67,7 +74,7 @@ def package_vars(version_file):
         "docopt >= 0.6.2",
         "mongo-db-from-config @ https://github.com/cisagov/mongo-db-from-config/tarball/develop#egg=mongo-db-from-config",
         "netaddr >= 0.7.10",
-        "python-dateutil >= 2.2",
+        "python-dateutil >= 2.2, < 2.8.1",
```

Comment:
Can you please add a comment in here about why we want to stay below 2.8.1?

Reply:
Ah, good point. This should be resolved in 76551ed